rmgpy/kinetics/surface.pyx

```diff
@@ -119,15 +132,31 @@ cdef class StickingCoefficient(KineticsModel):
         def __set__(self, value):
             self._T0 = quantity.Temperature(value)
 
+    property cov:
+        """The coverage dependence parameters."""
+        def __get__(self):
+            return self._cov
+        def __set__(self, value):
+            self._cov = {}
+            if value:
+                for species, parameters in value.items():
+                    for variable, x in parameters.items():
+                        if variable is 'E':
+                            parameters[variable] = quantity.Energy(x)
+                        else:
+                            parameters[variable] = quantity.Dimensionless(x)
+                    self._cov[species] = parameters
+
```

Comment:
Maybe we could add some type/value checks here? Should species be strings (SMILES str)?  Can `x` have any value (-inf,inf)?  Do we need default values for some of the parameters if they are not given? If `m` is not in value, parameters[variable] = default?

Reply:
Maybe we should check that the cov dep correction is very close to 1 at very low coverages with the given parameters?

Reply:
You'd only need to specify a default m or a if only `E` was given when entering coverage dependence in the entry. I _think_ I have it set up so that when you enter in your coverage dependencies, you must provide all 3 values or else you'll fail the database check, which means there should never be an instance when you need to assume a default `a` or `m`

Reply:
This code block (and the other instance of it below) could at least be written to enforce there are the correct 3 parameters. Would probably reduce looping and branching etc. too.
eg.
```python
        def __set__(self, value):
             self._coverage_dependence = {}
             if value:
                 for species, parameters in value.items():
                     processed_parameters = {'E': quantity.Energy(parameters['E']),
                                             'm': quantity.Dimensionless(parameters['m']),
                                             'a': quantity.Dimensionless(parameters['a'])}
                     self._coverage_dependence[species] = processed_parameters
```
You could add error handling if you want, but at least this will complain loudly if a parameter is missing, unexpected, or has the wrong units.